pandas/core/nanops.py

```diff
@@ -812,15 +812,21 @@ def unique1d(values):
     return uniques
 
 
-def _checked_add_with_arr(arr, b):
+def _checked_add_with_arr(arr, b, arr_nans=None, b_nans=None):
```

Comment:
why don't we make this signature more consistent maybe
``_checked_add_with_arr(a, b, a_mask=None, b_mask=None)``